packages/checkout/widgets-lib/src/widgets/sale/views/PayWithCoins.tsx

```diff
@@ -13,14 +13,25 @@ export function PayWithCoins() {
     sendTransactionSuccessEvent,
     sendFailedEvent,
     sendCloseEvent,
+    sendSuccessEvent,
   } = useSaleEvent();
-  const { execute, signResponse, executeResponse } = useSaleContext();
+  const {
+    execute, signResponse, executeResponse, signTokenIds,
+  } = useSaleContext();
   const executedTxns = executeResponse?.transactions.length || 0;
 
-  let loadingText = t('views.PAYMENT_METHODS.loading.ready');
+  let loadingText = [
```

Comment:
Hey @jhesgodi just wanted to show this option for getting back an array of keys..
t('views.PAYMENT_METHODS.loading.ready', { returnObjects: true })